extensions/common/azure/azure-blob-core/src/main/java/org/eclipse/edc/azure/blob/adapter/DefaultBlobAdapter.java

```diff
@@ -24,15 +27,27 @@
  * Implementation of {@link BlobAdapter} using a {@link BlockBlobClient}.
  */
 public class DefaultBlobAdapter implements BlobAdapter {
+    private final long blockSizeInMb;
+    private final int maxConcurrency;
+    private final long maxSingleUploadSizeInMb;
     private final BlockBlobClient client;
 
-    public DefaultBlobAdapter(BlockBlobClient client) {
+    public DefaultBlobAdapter(
+            BlockBlobClient client, Long blockSizeInMb, Integer maxConcurrency, Long maxSingleUploadSizeInMb) {
+        this.blockSizeInMb = blockSizeInMb;
+        this.maxConcurrency = maxConcurrency;
+        this.maxSingleUploadSizeInMb = maxSingleUploadSizeInMb;
         this.client = client;
     }
 
     @Override
     public OutputStream getOutputStream() {
-        return client.getBlobOutputStream(/* overwrite = */ true);
```

Comment:
I'm not really an expert on this SDK, but is the behavior we had with passing the `true` the same as passing the blob output stream options?